pcsx2/x86/newVif_Dynarec.cpp

```diff
@@ -347,8 +347,11 @@ _vifT __fi void dVifUnpack(const u8* data, bool isFill) {
 
 		u8*  startmem = VU.Mem + (vif.tag.addr & (vuMemLimit-0x10));
 		u8*  endmem   = VU.Mem + vuMemLimit;
-
+#if 0
 		if (likely((startmem + b->length) <= endmem)) {
+#else
+    if(0) {
+#endif
```

Comment:
This is meant to divert into the interpreter (assuming that it's shorter code that's easier to debug).